port/android/Makefile

```diff
@@ -185,17 +185,11 @@ ifeq ($(CLOUD),1)
 	TCP=1
 	IPV4=1
 	SAMPLES += cloud_server cloud_tests
+	#NATIVE_SAMPLES += cloud_linux cloud_tests
 endif
 
 OBT = onboarding_tool
 
-ifeq ($(CLOUD),1)
-	CFLAGS += -DOC_CLOUD
```

Comment:
We want to allow build cloud for android or it is not used at all?

Reply:
Duplicate, look above at lines 183-189.